clients/cfrestclient/operations/get_shared_domains_responses.go

```diff
@@ -33,7 +35,7 @@ func (o *GetSharedDomainsReader) ReadResponse(response runtime.ClientResponse, c
 		return result, nil
 
 	default:
-		return nil, runtime.NewAPIError("unknown error", response, response.Code())
+		return nil, baseclient.ProcessErrorResponse(response, consumer, o.formats)
```

Comment:
We need to make sure that nobody will forget to replace these lines of code in the future.